File: 1920-build-array-from-permutation/1920-build-array-from-permutation.cpp
class Solution {
public:
    vector<int> buildArray(vector<int>& nums) {
        int size = nums.size();
        vector<int> arr(size);

        for(int i = 0; i < size; i++) {
            arr[i] = nums[nums[i]];
        }

        return arr;
    }
};

File: 1920-build-array-from-permutation/1920-build-array-from-permutation.py
class Solution(object):
    def buildArray(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        size = len(nums)
        res = [0] * size

        for i in range(size):
            res[i] = nums[nums[i]]

        return res